uber/templates/dept_checklist/index.html

```diff
@@ -3,19 +3,44 @@
 {% block content %}
 
 <h2>Department Head Checklist for {{ attendee.full_name }}</h2>
+<ul class="nav nav-tabs">
```

Comment:
You committed these (AFAICT) completely unrelated changes in a commit message that simply says: "solve some comments to PR."

This is one of the reasons we get on each other's case about commit messages -- I have _no_ idea why you've changed anything here. I don't even know if you did it on purpose, because your commit message doesn't tell me anything about your intentions. I know it's tempting to cheap out on commit messages (we all do it from time to time), but especially if you're making big changes, please let us know why.